src/main/java/ua/com/juja/microservices/keepers/slackbot/model/request/KeeperRequest.java

```diff
@@ -0,0 +1,67 @@
+package ua.com.juja.microservices.keepers.slackbot.model.request;
+
+import lombok.Getter;
+import lombok.ToString;
+
+import ua.com.juja.microservices.keepers.slackbot.exception.WrongCommandFormatException;
+import ua.com.juja.microservices.keepers.slackbot.model.SlackParsedCommand;
+import ua.com.juja.microservices.keepers.slackbot.model.dto.UserDTO;
+
+/**
+ * @author Dmitriy Lyashenko
+ */
+@Getter
+@ToString
+public class KeeperRequest {
+
+    private String from;
+    private String uuid;
+    private String direction;
+
+    public KeeperRequest(String from, String uuid, String direction) {
+        this.from = from;
+        this.uuid = uuid;
+        this.direction = direction;
+    }
+
+    public KeeperRequest(SlackParsedCommand parsedCommand) {
+        this.from = parsedCommand.getFromUser().getUuid();
+        this.uuid = receiveToUser(parsedCommand).getUuid();
+        this.direction = receiveToDirections(parsedCommand);
+    }
+
+    private UserDTO receiveToUser(SlackParsedCommand slackParsedCommand) {
+
+        int userCount = slackParsedCommand.getUserCount();
+
+        if (userCount > 1) {
+            throw new WrongCommandFormatException(String.format("We found %d slack names in your command: '%s' " +
+                            " You can't make two Keepers on one direction.", slackParsedCommand.getUserCount(),
+                    slackParsedCommand.getText()));
```

Comment:
" You can't make two Keepers on one direction." - two keepers it is not correct, because in this case we can have more than two slackNames.